### ISSUE TITLE
Support for linked images

### ISSUE THREAD

**Issue by flavordaaave** [mientjan/react-native-markdown-renderer#24]
I have something like this in my markdown:

```
[![Linked image](http://example.com/image.png)](http://example.com/linkedpage.html)
```

Now I'm trying to have a switch in a custom renderer for `link` type:
```
  link: (node, children, parent, styles) => {
    const firstChild = node.children || [{}]
    if (firstChild.type === 'image') {
      return <Touchable>...</Touchable>
    } else {
      return (
        <Text
          key={node.key}
          style={styles.link}
          onPress={() => console.log('OPEN LINK: ', node.attributes.href)}>
          {children}
        </Text>
      )
    }
  },
```

But unfortunately all links are treated as inline text (see `src/lib/util/getIsInlineTextType.js` which is why I think the children array is always empty for links.

Any plans on adding support for that as well?

**Comment by mientjan** [comment]
Yes, this is a must have feature.

**Comment by mientjan** [comment]
fixed issue in branch https://github.com/mientjan/react-native-markdown-renderer/tree/feature/issue-33 will merge soon. could you test it? @flavordaaave

**Comment by mientjan** [comment]
released on v3.2.0
